### PR TITLE
Feat/be get each review by movie id user

### PR DESCRIPTION
**Pull request by YF91925352**
# Description
Create the endpoint to get specific review by moviesId and UID,which will be used to the user rating start and each user review part
# Jira task link
https://hackyourfuture-dk.atlassian.net/jira/software/projects/VIEW/boards/7?selectedIssue=VIEW-74
# Screenshots (if applicable)

### CODE REVIEW

File: packages/server/api/routes/uidreview.router.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+// controllers
+const uidreviewController = require('../controllers/uidreview.controller');
+
+router.get('/:id/:uid', (req, res) => {
```
Comment (itsgauravsaxena) [inline packages/server/api/routes/uidreview.router.js:7]: I would put it like 
/:id/uid/:uid just to make it bit readable

File: packages/server/api/routes/uidreview.router.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  uidreviewController
```
Comment (itsgauravsaxena) [inline packages/server/api/routes/uidreview.router.js:8]: Not sure if a new router is needed. Could you not put this inside moviecontroller or review controller?
Route should represent some sort of business object or entity

File: packages/server/api/controllers/uidreview.controller.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getReviewByUid = async (id, uid) => {
+  if (!id) {
```
Comment (itsgauravsaxena) [inline packages/server/api/controllers/uidreview.controller.js:5]: You could also do a check if id is a number and not string. Like isNan

**Review by itsgauravsaxena — CHANGES_REQUESTED** [review]
Added some comments for BE change.
Please get some reviews from front-end reviewers too

**Comment by YF91925352** [comment]
> Added some comments for BE change. Please get some reviews from front-end reviewers too

Hi~the frontend reviewer has approved the style changes in other branch.This branch is created based on the main,byt when I git pull origin main,the other branches will be updated together

> Added some comments for BE change. Please get some reviews from front-end reviewers too

File: packages/server/api/routes/uidreview.router.js (hunk 1 above)
Comment (YF91925352) [reply packages/server/api/routes/uidreview.router.js:7]: I have changed it to /:id/uid/:uid

File: packages/server/api/controllers/uidreview.controller.js (hunk 3 above)
Comment (YF91925352) [reply packages/server/api/controllers/uidreview.controller.js:5]: OK! I change it to isNaN()

File: packages/server/api/routes/uidreview.router.js (hunk 2 above)
Comment (YF91925352) [reply packages/server/api/routes/uidreview.router.js]: I have change it to the review controller

**Comment by YF91925352** [comment]
> Added some comments for BE change. Please get some reviews from front-end reviewers too

Thx for your comments!I have changed them.Could you please have a look,thx a lot?

**Review by itsgauravsaxena — APPROVED** [review]

**Pull request merged**